test/test_advection.cpp

```diff
@@ -71,15 +68,16 @@ TEST_CASE("Advection Equation")
     NeoFOAM::Database db;
     fvcc::FieldCollection& fieldCollection = fvcc::FieldCollection::instance(db, "fieldCollection");
 
-    NeoFOAM::Executor exec = GENERATE(
-        NeoFOAM::Executor(NeoFOAM::SerialExecutor {}),
-        NeoFOAM::Executor(NeoFOAM::CPUExecutor {}),
-        NeoFOAM::Executor(NeoFOAM::GPUExecutor {})
+    NeoFOAM::Executor exec = GENERATE(NeoFOAM::Executor(NeoFOAM::SerialExecutor {})
+                                      // NeoFOAM::Executor(NeoFOAM::CPUExecutor {}),
+                                      // NeoFOAM::Executor(NeoFOAM::GPUExecutor {})
```

Comment:
Add either a TODO: or FIXME note

```suggestion
    // TODO: add executors CPU/GPU again
    NeoFOAM::Executor exec = GENERATE(NeoFOAM::Executor(NeoFOAM::SerialExecutor {})
                                      // NeoFOAM::Executor(NeoFOAM::CPUExecutor {}),
                                      // NeoFOAM::Executor(NeoFOAM::GPUExecutor {})
```